boards/atxmega-a1-xpro/include/gpio_params.h

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright (C) 2021 Gerson Fernando Budke
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     boards_atxmega-a1-xpro
+ * @{
+ *
+ * @file
+ * @brief       Configuration of SAUL mapped GPIO pins
+ *
+ * @author      Gerson Fernando Budke <nandojve@gmail.com>
+ */
+
+#ifndef GPIO_PARAMS_H
+#define GPIO_PARAMS_H
+
+#include "board.h"
+#include "saul/periph.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/**
+ * @brief   GPIO configuration
+ */
+static const saul_gpio_params_t saul_gpio_params[] =
+{
+    {
+        .name = "SW0",
+        .pin = BTN0_PIN,
+        .mode = BTN0_MODE,
+        .flags = SAUL_GPIO_INVERTED,
+    },
+    {
+        .name = "LED0 (Yellow)",
+        .pin = LED0_PIN,
+        .mode = LED0_MODE,
+        .flags = (SAUL_GPIO_INVERTED | SAUL_GPIO_INIT_SET),
```

Comment:
```suggestion
        .flags = SAUL_GPIO_INVERTED,
```

This is currently turning the LED on, but only when `saul_auto_init` is used. IMO it would be better to have auto_init behavior consistent regardless of whether SAUL is used. I think that most other boards also just leave the LED turned off during boot.